parser.cpp

```diff
@@ -1185,9 +1194,9 @@ namespace Sass {
     { return new (ctx.mem) Null(pstate); }
 
     if (lex< identifier >()) {
-      String_Constant* str = new (ctx.mem) String_Constant(pstate, lexed);
+      String_Constant* str = new (ctx.mem) String_Quoted(pstate, lexed);
```

Comment:
`String_Constant` will not be unquoted, never. This way we can preserve escape sequences (which is not the case if you unquote/quote it). To be honest it's also quite confusing to me and I don't really know if I have confuses me too. But be aware, that this PR has no real implementation for `String_Quoted`. This will be added later. Maybe this helps to clarify it a bit more: [ast.hpp] [1] and [inspect.cpp] [2]. A `String_Quoted` will be added with quotes to the output if `quote_mark` is set (has no effect on `String_Constant`).

[2]: https://github.com/mgreter/libsass/blob/59d811967e3530c2c6125a8628f1fa3cc8793968/ast.hpp#L1393
[1]:  https://github.com/mgreter/libsass/blob/59d811967e3530c2c6125a8628f1fa3cc8793968/inspect.cpp#L570